espnet2/bin/asr_inference_streaming.py

```diff
@@ -0,0 +1,558 @@
+#!/usr/bin/env python3
```

Comment:
I think this inference design may need some discussion.
It would be great if @hfujihara, @eml914, @b-flo, @kamo-naoyuki share your ideas given your use case.